### DISCUSSION TITLE
View files in a PR collapsed not expanded

### DISCUSSION THREAD

**Discussion by gennadiycivil** [community/community#21557] [New to GitHub]
<p>I have to revew PRs … many PRs.</p>
<p>When I cick to see the changed files I see all of the files and changes  _expanded with changes _which is inconvenient and has been an issue for me.</p>
<p>How do I see the list of files collapsed so I can pick the files I actually want to see?</p>

**Comment by tzmanics** [comment, marked as answer]
<p>Hi! :wave: So, I’m not sure if you can have this as a default but you CAN collapse the files you don’t need to see that have been changed by clicking the arrow at the top right corner of each file (see pic of random PR i found for an example :grinning:).</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/1/195fd4a2c4356b115f3d43c0781b5f939ba3397b.jpeg" data-download-href="/uploads/short-url/3CtfG7y8EJe6D3AxDemL9Xu3kkH.jpg?dl=1" title="commitfiles_LI.jpg" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/3611928/181065423-7fbf3d4b-dda7-4d02-9066-007b5d2f779e.jpg" alt="commitfiles_LI.jpg" data-base62-sha1="3CtfG7y8EJe6D3AxDemL9Xu3kkH" width="690" height="196" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/1/195fd4a2c4356b115f3d43c0781b5f939ba3397b_2_690x196.jpg, https://aws1.discourse-cdn.com/github/optimized/2X/1/195fd4a2c4356b115f3d43c0781b5f939ba3397b_2_1035x294.jpg 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/1/195fd4a2c4356b115f3d43c0781b5f939ba3397b_2_1380x392.jpg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/1/195fd4a2c4356b115f3d43c0781b5f939ba3397b_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">commitfiles_LI.jpg</span><span class="informations">2702×770 265 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>Hope that helps!</p>

**Comment by lsloan** [comment]
<p>Good request!  I was thinking of that recently, too.  I’d like the option of all the changes starting out as collapsed, then I could expand the ones I need to see.  There should also be controls to expand all or collapse all changes.  (Maybe even a toggle all button that expands all collapsed changes and vice versa.)</p>

**Comment by lsloan** [comment]
<p>IMNSHO, this is not a good solution.  My colleague opened a PR which contained more than thirty changes.  Five were changes I was interested in and the others were new, large JSON files that were added to the repo.  During the review, the files I need to see weren’t located contiguously, so I had to painstakingly… click… each… collapse… button… as… you… described… one… by… one.  😛</p>
<p>GH needs to have controls in PR code reviews for collapse all, expand all, and probably toggle all.  There should also be an setting to start reviews with all changes collapsed.</p>

**Comment by nadiajoyce** [comment]
<p><a class="mention" href="/lsloan">@lsloan</a> This is great feedback. We’re always working to improve GitHub, and we consider every suggestion we receive. However, we currently are asking that feature requests be submitted via our contact form <a href="https://github.com/contact" rel="nofollow noopener">here</a>. We are working on introducing a feature to the Community Forum that will allow users to submit feature requests right here, but for now, we are continuing to log and consider feature requests via email. </p>

**Comment by lsloan** [comment]
<p>Gotcha!  This forum _feels_ like the place for this type of discussion and feedback, so I bet you’ll send many, many messages directing users to that contact page:</p>
<p>* <a href="https://github.com/contact" rel="nofollow noopener">https://github.com/contact</a></p>
<p>Let’s hope a GH issue reporter appears here (or better yet, in GH) soon.</p>
<p>PS: Have you seen:</p>
<p>* <a href="https://github.com/dear-github/dear-github" rel="nofollow noopener">https://github.com/dear-github/dear-github</a></p>
